analyzer/restapi-plugin/src/main/resources/application.properties

```diff
@@ -4,15 +4,13 @@ server.tomcat.connection-timeout=60000
 server.port=8080
 server.tomcat.threads.max=20
 kb.user=${db.user}
-kb.java.url=${db.java.url}
-kb.python.url=${db.python.url}
-kb.c.url=${db.c.url}
-kb.maven.depgraph.path=${dg.maven.path}
-kb.pypi.depgraph.path=${dg.pypi.path}
-kb.debian.depgraph.path=${dg.debian.path}
+kb.url=${db.url}
+kb.depgraph.path=${dg.path}
 kb.graphdb.path=${gd.path}
 kb.vulnchains.path=${vc.path}
 lima.rcg.url=${rcg.url}
+dependency.resolver.address=${dep.api}
```

Comment:
What is this variable supposed to contain?

Reply:
Hello, this variable contains the address of the dependency resolver microservices that have been deployed on monster: https://github.com/fasten-project/k8s-deployments/tree/master/fasten/resolvers
For the docker compose, a similar deployment with that of the stitching microservices (https://github.com/fasten-project/fasten-docker-deployment/pull/21) should take place in order to deploy the rest-api